viewer/src/components/BoardAction.vue

```diff
@@ -73,7 +82,16 @@ export default class BoardAction extends Vue {
   }
 
   get faded() {
-    return !this.$store.state.gaiaViewer.data.boardActions[this.action];
+    return this.planet != null;
+  }
+
+  get planet(): Planet {
+    const data = this.$store.state.gaiaViewer.data;
+    const player = data.boardActions[this.action];
+    if (player != null && player !== PlayerEnum.Player5) { // Player5 is for converted old games
```

Comment:
should also handle the boolean situation. For example when viewing completed games.